packages/x-components/src/x-modules/facets-next/store/__tests__/getters.spec.ts

```diff
@@ -0,0 +1,194 @@
+import { Filter } from '@empathyco/x-types-next';
+import { createLocalVue } from '@vue/test-utils';
+import Vuex, { Store } from 'vuex';
+import {
+  createNextEditableNumberRangeFilter,
+  createNextHierarchicalFilter,
+  createNextNumberRangeFilter,
+  createRawFilter,
+  createNextSimpleFilter
+} from '../../../../__stubs__/filters-stubs.factory';
+import { ActionsDictionary, XActionContext } from '../../../../store/actions.types';
+import { MutationsDictionary } from '../../../../store/mutations.types';
+import { arrayToObject } from '../../../../utils/array';
+import { Dictionary } from '../../../../utils/types';
+import { facetsNextXStoreModule } from '../module';
+import {
+  FacetsNextActions,
+  FacetsNextGetters,
+  FacetsNextMutations,
+  FacetsNextState
+} from '../types';
+import { resetFacetsStateWith } from './utils';
+
+/** Type safe single module store. */
+type SafeStore<
```

Comment:
This was my firefighter morning idea 🤣. I realised we already have kinda a type safe store, so we can avoid having to cast, or to use the actions/getters/mutations keys as variables.